package.json

```diff
@@ -24,8 +24,12 @@
     "babel-jest": "^26.0.1",
     "eslint": "^6.8.0",
     "eslint-config-plazide": "^0.1.6",
+    "faunadb": "^3.0.1",
```

Comment:
When I tried testing my changes locally via `npm link`, I ran into "module not found" errors for the `faunadb` dependency so I added this to dev dependencies.

I also listed `faunadb` is a peer dependency because I think it should be listed, but let me know if you disagree.